pgx-pg-sys/include/pg11.h

```diff
@@ -72,6 +72,7 @@ Use of this source code is governed by the MIT license that can be found in the
 #include "optimizer/planner.h"
 #include "optimizer/restrictinfo.h"
 #include "optimizer/tlist.h"
+#include "parser/analyze.h"
```

Comment:
Are there any C macros from this header that you want to bind?

Reply:
I don't think so. Why do you ask? I just added this to get the definitions of `pgx_post_parse_analyze` and `pgx_post_parse_analyze_type`

Reply:
Just making sure since C macros aren't bound by bindgen.